elements/pfe-tabs/src/pfe-tabs.js

```diff
@@ -419,7 +423,7 @@ class PfeTabs extends PFElement {
 
     event.preventDefault();
 
-    this.selectedIndex = this._getTabIndex(newTab) || 0;
+    this.selectedIndex = this._getTabIndex(newTab);
```

Comment:
Centralize the fallback behavior of the tab index.  More resilient.  Good stuff!